tilemap/Map.cs

```diff
@@ -0,0 +1,21 @@
+using Godot;
+using System;
+
+public class Map : Node2D
+{
+    // Declare member variables here. Examples:
+    // private int a = 2;
```

Comment:
this can be removed 😄

Reply:
[21bd372](https://github.com/fourlastor-jams/game-off-2022/pull/13/commits/21bd372b22ee0250ea70588a63e39dc9e7448937) :see_no_evil: